xonsh/environ.py

```diff
@@ -575,6 +579,12 @@ def xonshconfig(env):
     'XONSH_STORE_STDOUT': VarDocs(
         'Whether or not to store the stdout and stderr streams in the '
         'history files.'),
+    'XONSH_TRACEBACK_LOGFILE': VarDocs(
+        'Specifies a file to store the traceback log to, regardless of whether '
+        'XONSH_SHOW_TRACEBACK has been set. Its value must be a writable file '
+        'or None / the empty string if traceback logging is not desired. '
+        'Logging to a file is not enabled by default.'
+    )
     }
```

Comment:
@mitnk Hope it is ok now. Interestingly, pylint did not complain about the line with the leading spaces.